system-config-printer.py

```diff
@@ -545,6 +570,11 @@ def __init__(self):
 
 
         self.PrintersWindow.show()
+        self.builder = Gtk.Builder()
+        self.service_browser = AvahiServiceBrowser()
```

Comment:
Shouldn't these three lines be guarded by AVAHI_AVAILABLE as well? We call objects from Avahi at the method `run()`.